plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -57,6 +57,7 @@
     aliases: [ account_id ]
     description:
       - ID of account to delete/modify
+      - Can also be used in account creation to work around vendor issues where the ID of the new user is required in the POST request
```

Comment:
We might need to PEP8ify this to be consistent with the rest of the file, but otherwise looks good to me.